server/jetstream_test.go

```diff
@@ -11647,6 +11631,14 @@ func TestJetStreamSourceBasics(t *testing.T) {
 		return nil
 	})
 
+	m, err := js.GetMsg("MS", 1)
+	if err != nil {
```

Comment:
requie_NoError(t, err)

Reply:
Done, since this pattern was present in many places in the file I made it consistent and replaced with require_NoError in all these other places in the file as well